testflinger/__init__.py

```diff
@@ -157,17 +157,21 @@ def setup_mongodb(application):
     mongo_pass = os.environ.get("MONGODB_PASSWORD")
     mongo_db = os.environ.get("MONGODB_DATABASE")
     mongo_host = os.environ.get("MONGODB_HOST", "mongo")
+    mongo_uri = os.environ.get("MONGODB_URI")
     if not application.config.get("MONGO_URI") and not (
```

Comment:
Why is it called “MONGO_URI”, if we use “MONGO**DB**_URI”? Is it reasonable to change the value name in config?

Reply:
I agree it's inconsistent. I'm not sure of the impact of changing that in application config, though

Reply:
I guess we need @plars to take a look at this

Reply:
The difference is intentional here in order to use the same naming convensions as the modules and containers where the variables get used. MONGO_URI is the name of the config key that PyMongo will try to use from flask, so I thought it best to keep that as the same name for the environment variable.  This is done for similar reasons with MONGODB_{PASSWORD, DATABASE, ...} due to the expected vars in the mongodb container we are using.

If it makes you feel better, you will probably only ever use either MONGO_URI -or- the other MONGODB_* vars but not both at the same time.

Reply:
Should MONGO_URI name also be used in these changes instead of MONGODB_URI? Or it looks fine as is?

Reply:
No, we don't need to manually check os.environ for that, because it will be loaded into application.config from [this ilne](https://github.com/canonical/testflinger/blob/a62184dd91a693019dd6e63749e8cd3622a740ee/testflinger/__init__.py#L59)